cosmo_tester/test_suites/test_blueprints/resources_available_test.py

```diff
@@ -0,0 +1,44 @@
+########
+# Copyright (c) 2014 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+import requests
+from requests.exceptions import ConnectionError
+
+from cosmo_tester.framework.git_helper import clone
+from cosmo_tester.framework.testenv import TestCase
+
+
+class ResourcesAvailableTest(TestCase):
+
+    repo_url = ('https://github.com/cloudify-cosmo/'
+                'cloudify-nodecellar-example.git')
+
+    def test_resources_available(self):
```

Comment:
Can you please also add a test that verifies the resource is still available via the correct file server port (53229)?